addons/web/static/tests/views/view_dialogs_tests.js

```diff
@@ -243,6 +246,126 @@ QUnit.module('Views', {
         parent.destroy();
     });
 
+    QUnit.test('SelectCreateDialog cascade x2many in create mode', function (assert) {
+        assert.expect(5);
+
+        this.data.product = {
```

Comment:
i think you can put these data in the main demo data for this module. it should not interfere with other tests, it will make them usable in the future tests